consensus/enclave/api/src/lib.rs

```diff
@@ -246,6 +247,22 @@ pub struct FormBlockInputs {
     pub mint_txs_with_config: Vec<(MintTx, MintConfigTx, MintConfig)>,
 }
 
+/// The schema expected to be encrypted for client_tx_propose_v2
+#[derive(Clone, Message)]
+pub struct ClientProposeTxRequest {
```

Comment:
This is the request object that clients use to propose Tx's in the new `client_propose_tx_v2`.